imagedephi/main.py

```diff
@@ -153,35 +167,110 @@ def run(
     verbose,
     log_file,
     index,
+    command_file: Path,
 ):
     """Perform the redaction of images."""
     params = _check_parent_params(ctx, profile, override_rules, recursive, quiet, verbose, log_file)
     if params["verbose"] or params["quiet"] or params["log_file"]:
         set_logging_config(params["verbose"], params["quiet"], params["log_file"])
+    command_params = {}
+    command_inputs: list[Path] = []
+    command_output: Path
+    cf_override_rules: Path | None = None
+    cf_profile: str = ""
+    # cf_rename will only be checked if rename is false so default should be false.
+    cf_rename: bool = False
+    cf_recursive: bool = False
+    cf_index: int = 1
+    if command_file:
+        if command_file.suffix not in [".yaml", ".yml"]:
+            with command_file.open() as f:
+                command_inputs = [
+                    Path(line) for line in f.read().splitlines() if Path(line).exists()
+                ]
+            if output_dir is None:
+                output_dir = Path.cwd()
+        else:
+            with command_file.open() as f:
+                command_params = yaml.safe_load(f)
+                try:
+                    CommandFile.model_validate(command_params)
+                except ValidationError:
+                    click.echo(
```

Comment:
I wonder if we should support a different command line flag to always treat the file as a list (and then not get this warning).

Reply:
In this scenario would we want to still prompt a warning like this if that command line flag isn't supplied?

Reply:
My though was that if we had two flags then the current `-c` would just print an error and stop and the new flag (maybe `-f` and `--filelist`) would expect just a list.

Reply:
added in 17e09633d7a9e7d65761933198c444070a101731